patricks.php

```diff
@@ -11,7 +11,7 @@
 require("PulseAudio.API.php");
 require("pa.cli.php");
 
-$PA = new PulseAudio(`LANG=en_US.UTF-8 pactl list`, `LANG=en_US.UTF-8 pactl stat`);
+$PA = new PulseAudio(`env LC_ALL=C pactl list`, `env LC_ALL=C pactl stat`);
```

Comment:
@dimaw Thanks! You make my day! Recently several hours spend to found right locale setting for `pactl`. 🚀